core/services/pipeline/task.bridge.go

```diff
@@ -18,13 +21,17 @@ type BridgeTask struct {
 	Name              string `json:"name"`
 	RequestData       string `json:"requestData"`
 	IncludeInputAtKey string `json:"includeInputAtKey"`
+	Async             string `json:"async"`
```

Comment:
I think we'll need some new validation in the ValidatedXX methods which should error if someone tries to create an {OCR,FM,VRF} spec with an async job in it